qutip/tests/test_bloch.py

```diff
@@ -361,3 +361,10 @@ def test_vector(self, vector_kws, fig_test, fig_ref):
             vector_kws = [vector_kws]
         self.plot_vector_test(fig_test, copy.deepcopy(vector_kws))
         self.plot_vector_ref(fig_ref, copy.deepcopy(vector_kws))
+
+    def test_representations(self):
+        b = Bloch()
+        assert 'Bloch' in repr(b)
+        assert str(b).startswith('Bloch data:')
+        repr_svg = b._repr_svg_()
+        assert repr_svg is None or isinstance(repr_svg, str)
```

Comment:
As @AGaliciaMartinez pointed out, this call to `_repr_svg_` exits early and returns `None` since ipython isn't installed during the tests.